src/Zicht/Bundle/UrlBundle/Aliasing/Listener.php

```diff
@@ -21,11 +21,26 @@
  */
 class Listener
 {
+    const SLASH_SUFFIX_IGNORE = 'ignore';
```

Comment:
I'm still not very happy with the naming here. "Ignore" might suggest the behaviour of "accept", which is to also take the _wrong_ URL (with `/` suffix) and just continues to render a response. "ignore" is how it is now, trying to find a URL alias **including** that slash, which isn't there most likely, so the listener does nothing and finally resulting in a 404 error.

Any suggestions for not doing anything with a possible slash suffix (and let it resolve by other means)?

- "deflect"?
- "no-action"?
- "pass"?
- `null`?
- "none"?

Reply:
`abstain` ? Is that what is actually happening?

Reply:
Great one!

Reply:
Wat gebeurd er exact wanneer 'ignore/abstain' of 'accept' gekozen is als optie?  Want voor mijn gevoel is het nog steeds niet duidelijk wat er gebeurd op basis van die woorden.

Reply:
Nee, daarom struggle ik er ook wat mee... 'ignore/abstain' houdt het bij de huidige werking. Alleen een exacte match van een URL pad: `/contact`. Probeer je `/contact/`, dan kan ie niks vinden en doet ie ook niks. Wat dan hoogstwaarschijnlijk resulteert in een 404 error.